classification/support/view_ic_over_movie_interactively.m

```diff
@@ -10,15 +10,44 @@
 active_frame_padding = padding; % Use 100 for 20 Hz movie
 time_window = 10; % Width of running window
 
+%Custom Scaling for movie display
+%-----------------------------------------------------------
+%CLim parameters for imagesc(). The movie is scaled to [0,1], so the most
+%natural choice for [clim_lower,clim_upper] = [0,1]. However, it might be
+%set to values in the neighborhood to have darker or higher-contrast image.
+clim_lower = 0;
+clim_upper = 1.4;
+
+%the upper and lower quantiles to be used in the calculation of the
+%brigthest and the darkest pixel values in the movie. The aim is to 
+%mitigate the effects of possible outlier pixels.
+quant_upper = 0.99;
+quant_lower = 0.85;
+
+% Calculate the 'darkest' pixel(ZLow) and the 'brightest' pixel(ZHigh)
+[height,width,~] = size(movie);
+maxVec = reshape(max(movie,[],3),height*width,1);
+minVec = reshape(min(movie,[],3),height*width,1);
+
+threshUpBright = quantile(maxVec,quant_upper);
+threshDownBright = quantile(maxVec,quant_lower);
+
+threshUpDark = quantile(minVec,quant_upper);
+threshDownDark = quantile(minVec,quant_lower);
+
+ZHigh = mean(maxVec(maxVec>threshDownBright & maxVec<threshUpBright));
+ZLow = mean(minVec(minVec>threshDownDark & minVec<threshUpDark));
+

```

Comment:
Performing the above set of operations _for each IC_ is expensive and has considerably slowed down the load time of each plot.